### ISSUE TITLE
Cannot identify stateless component

### ISSUE THREAD

**Issue by damian** [reactjs/react-docgen#168]
The following bug is present in react-docgen v2.13.0.

My component looks like this:

```
const Visibility = ({ isVisible, children }: Props) => (
  isVisible ? React.Children.only(children) : null
);
export default Visibility;
```

When I run `react-docgen src/components/visibility/visibility.js --pretty` over it I receive the following error:

```
Error with path "src/components/visibility/visibility.js": Error: No suitable component definition found.
Error: No suitable component definition found.
```

I thought it may have been due to issue #107 so I refactored this into a standard if block, which returned the same error.

If you have any thoughts about how to get around this in the short term, while the underlying issue is being fixed it'd be much appreciated :)

**Comment by fkling** [comment]
We are simply not recognizing `React.Children.only`. In that case, there is no workaround, sorry 😕 

The quickest way to get this fixed would be open a PR that adds a check for `React.Children.only` here: https://github.com/reactjs/react-docgen/blob/61a3c1c02a20855ba4bac578ca02ad1986d5e1a9/src/utils/isStatelessComponent.js#L35 😉 

Solving #107 needs some more work though...

**Comment by fkling** [comment]
Fixed via 34ab2aef215e2c557a25e952540ed868334dd349 and released as part of v2.14.0 and v3.0.0-beta3.
